examples/src/main/java/com/cloudera/nav/sdk/examples/managed/StetsonScript.java

```diff
@@ -51,13 +48,7 @@ public StetsonScript(String namespace) {
    */
   @Override
   public String generateId() {
-    if (!Strings.isNullOrEmpty(getPigOperation().getIdentity())) {
-      return CustomIdGenerator.generateIdentity(getNamespace(),
-          getPigOperation().getIdentity());
-    } else {
-      return CustomIdGenerator.generateIdentity(getNamespace(),
-          UUID.randomUUID().toString());
-    }
+    return CustomIdGenerator.generateIdentity(getNamespace(), getName());
```

Comment:
We should add some comments here and in other custom classes that these fields must uniquely identify the entity